src/bindgen/ir/annotation.rs

```diff
@@ -53,6 +55,10 @@ impl AnnotationSet {
         self.must_use && config.language != Language::Cython
     }
 
+    pub(crate) fn deprecated(&self, config: &Config) -> bool {
```

Comment:
let's make this `fn deprecated_node(&self, config: &Config) -> Option<&str>`. That allows the `unwrap()`s elsewhere.